content/md/user-management/manage-your-users-via-import-export.md

```diff
@@ -0,0 +1,36 @@
+---
+id: manage-your-users-via-import-export
+themes: user-management
+title: Manage your users via import and export
+popular: false
+related: what-is-a-user-group, what-is-a-user, access-rights-on-products, access-rights-on-assets, access-rights-on-imports-exports
+---
+
+# Import/export your users
+
+Let's say that you have to initialize your PIM, because your PIM project just starts or because you just migrate from a Flexibility offer to a Serenity one. Configure the users and their permissions can be a heavy task through the PIM UI, if you have a lot of users to set up. That's why we offer you the possibility to **import/export your users** and their information, such as their user groups and roles, their UI locale, their default category tree, etc.
```

Comment:
```suggestion
Let's say that you have to initialize your PIM, because your PIM project just starts or because you're migrating from a Flexibility offer to a Serenity one. Configuring the users and their permissions can be a heavy task through the PIM UI, if you have a lot of users to set up. That's why we offer you the possibility to **import/export your users** and their information, such as their user groups and roles, their UI locale, their default category tree, etc.
```